itest/rust/src/rect2_test.rs

```diff
@@ -0,0 +1,109 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+use std::fmt::Debug;
+
+use crate::itest;
+use godot::{
+    builtin::{Rect2, RectSide, Vector2},
+    prelude::inner::InnerRect2,
+};
+
+#[itest]
+fn rect2_equiv_unary() {
```

Comment:
Hey! This test seems to fail due to floating-point arithmetic.
Idk if it should be prefered to solve floating-point or to use `is_equal_approx`.